cirq/ops/gate_operation.py

```diff
@@ -167,8 +167,18 @@ def _unitary_(self) -> Union[np.ndarray, NotImplementedType]:
             return getter()
         return NotImplemented
 
-    def _commutes_(self, other: Any, atol: float) -> Union[bool, NotImplementedType, None]:
-        return self.gate._commutes_on_qids_(self.qubits, other, atol=atol)
+    def _commutes_(
+        self, other: Any, atol: Union[int, float] = 1e-8
+    ) -> Union[bool, NotImplementedType, None]:
+
+        commutes = self.gate._commutes_on_qids_(self.qubits, other, atol=atol)
+        if commutes is not NotImplemented:
+            return commutes
+
+        getter = getattr(super(), '_commutes_', None)
```

Comment:
I have two questions: 
1. What motivated the changes in this method? 
2. Also, why do we need to check for the existence of `_commutes_` on `super()`? We _know_ that `cirq.Operation` implements `_commutes_` now, as we just implemented it...